src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -970,28 +746,12 @@ private bool TryScanInterpolatedString(ref TokenInfo info)
         {
             Debug.Assert(TextWindow.PeekChar() == '$');
 
-            if (TextWindow.PeekChar(1) == '$')
+            if (TextWindow.PeekChar(1) is '$' or '@' or '"')
             {
                 // $$ - definitely starts a raw interpolated string.
```

Comment:
```suggestion
                // $$ - definitely starts a raw interpolated string.
                // $@ - definitely starts an interplated string.
                //
                // This will also match for $@@.  This is an error case when the user thinks they can mix verbatim and raw
                // interpolations together.  This will be properly handled in ScanInterpolatedStringLiteral
```